### PR TITLE
New directory for github issues and simple tests for parsing output 

### PR DESCRIPTION
**Pull request by thefishua**
In this pull request there are tests for parsing output that represent the specification attached below for a github issue with some stub function code and directory setup for github issues

Github Issue Example

### CODE REVIEW

File: cmd/internal/main.go [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package main
+
+func main()  {
```
Comment (The-TT-Hacker) [inline cmd/internal/main.go:3]: we dont need this

File: cmd/internal/parse.go [hunk 2]
```diff
@@ -0,0 +1,161 @@
+package main
```
Comment (The-TT-Hacker) [inline cmd/internal/parse.go:1]: I would put this in a internal/restler directory. there shouldn't be an internal directory in the cmd directory

**Comment by The-TT-Hacker** [comment]
if you move the issue parsing code to the internal directory this PR doesnt need the web-server directory where the api code was moved

**Comment by thefishua** [comment]
Will do going to refactor the code aswell

**Pull request closed without merging**